### PR TITLE
Support wallet connect

### PR DESCRIPTION
**Pull request by alexkeating**
## GitHub Issue

#169 

## Changes

Add global to app

## Checks

Before making your PR, please check the following:

- [ ] Critical lint errors are resolved
- [ ] App runs locally
- [ ] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: apps/summon-app/index.html [hunk 1]
```diff
@@ -8,6 +8,11 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
+    <script>
```
Comment (skuhlmann) [inline apps/summon-app/index.html:11]: The only thing that worries me about this is that we need all clients using this package to add this to their app UI.

If this is the only way to solve - then maybe we need to get some 'Getting Started' docs for this package and ensure this is communicated. Thoughts @jonathanprozzi ?

File: libs/daohaus-connect-feature/src/utils/defaults.ts [hunk 2]
```diff
@@ -11,16 +11,15 @@ export const supportedNetworks = addKeychain(
 
 const providerOptions: IProviderOptions = {
   // TODO
```
Comment (skuhlmann) [inline libs/daohaus-connect-feature/src/utils/defaults.ts:13]: TO-DONE

Comment (alexkeating) [reply libs/daohaus-connect-feature/src/utils/defaults.ts]: Updated

File: apps/summon-app/index.html (hunk 1 above)
Comment (alexkeating) [reply apps/summon-app/index.html:11]: Yes, it will be necessary for all vite apps. We should document it. Let me know where and I can add something

Comment (skuhlmann) [reply apps/summon-app/index.html:11]: oh ok. just for vite. cool that alleviate my concern.

**Review by skuhlmann — APPROVED** [review]

File: apps/summon-app/index.html (hunk 1 above)
Comment (jonathanprozzi) [reply apps/summon-app/index.html:11]: I was wondering that too since this is within our package. We can be sure to add this into the docs

**Review by jonathanprozzi — APPROVED** [review]
Nice. Wonder if we want to add the docs to our Wiki or to the package README? I can add to the README first and the Wiki but I'm wondering at what point we may also want a dedicated dev docs site. Something for us to keep on our radar @skuhlmann @alexkeating @earth2travis

**Pull request merged**